bittensor/utils/weight_utils.py

```diff
@@ -20,6 +20,7 @@
 # DEALINGS IN THE SOFTWARE.
 
 import hashlib
+import logging
```

Comment:
it is preferable to use `bittensor.logging` instead of `logging`

Reply:
👍  Will make this change in staging as well